src/os/web.rs

```diff
@@ -169,12 +177,18 @@ impl Target for Os {
         let begin = if let Some(b) = string.find('(') {
             b
         } else {
-            return Platform::Unknown("Unknown".to_string());
+            return Platform::Unknown(
+                "Failed to parse platform: Missing '(' in user agent"
+                    .to_string(),
+            );
         };
         let end = if let Some(e) = string.find(')') {
             e
         } else {
-            return Platform::Unknown("Unknown".to_string());
+            return Platform::Unknown(
+                "Failed to parse platform: Missing ')' in user agent"
+                    .to_string(),
+            );
```

Comment:
Sorry, I didn't catch this in my first review, but I think it would be more useful to just dump the full user agent instead of the `Failed to parse platform:` message.  (so returning `Platform::Unknown(string)`